src/ProjectTemplates/Web.ProjectTemplates/content/StarterWeb-CSharp/Views/Shared/_Layout.cshtml

```diff
@@ -5,7 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>@ViewData["Title"] - Company.WebApplication1</title>
     <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.min.css" />
-    <link rel="stylesheet" href="~/css/site.css" />
+    <link rel="stylesheet" href="~/Company.WebApplication1.styles.css" />
```

Comment:
This is interesting. In the Blazor templates, we still have the global CSS files as well as the scoped ones. I see that you've been able to make the global one unnecessary here. Is it possible that developers will still want to have a global non-scoped CSS file too?

Reply:
Good call!

I've brought back the non-scoped CSS file and moved the `html`/`body` style to it. This will help indicate to developers that non-scoped CSS files are still sensible to use in some scenarios.